backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/CustomizedResponseEntityExceptionHandler.kt

```diff
@@ -37,4 +38,13 @@ class CustomizedResponseEntityExceptionHandler : ResponseEntityExceptionHandler(
         val exceptionResponse = ExceptionResponse(usernameTaken.message.orEmpty())
         return ResponseEntity<ExceptionResponse>(exceptionResponse, HttpStatus.CONFLICT)
     }
+
+    @ExceptionHandler(PlayerDoesNotExistException::class)
+    fun handleUsernameTakenException(
+        playerDoesNotExist: PlayerDoesNotExistException,
+        request: WebRequest
+    ): ResponseEntity<ExceptionResponse> {
+        val exceptionResponse = ExceptionResponse(playerDoesNotExist.message.orEmpty())
+        return ResponseEntity<ExceptionResponse>(exceptionResponse, HttpStatus.NOT_ACCEPTABLE)
```

Comment:
is 406 a good response for that... I don't want to reply with 404 as it's used for no such room inside the same endpoint

Reply:
I'd go with 404 and exception-specific response (you can use Exception's message to instantiate it during creation of Exception), but if you feel it's too much effort then it's acceptable...